cmd/gateway-gcs-layer.go

```diff
@@ -153,7 +161,20 @@ type gcsGateway struct {
 }
 
 // newGCSGateway returns gcs gatewaylayer
-func newGCSGateway(endpoint string, projectID, secretKey string, secure bool) (GatewayLayer, error) {
+func newGCSGateway(args []string) (GatewayLayer, error) {
+	if len(args) != 1 {
+		return nil, fmt.Errorf("ProjectID expected")
+	}
+
+	profile := "default"
+	// TODO: don't know where to set profile yet
+	_ = profile
```

Comment:
I think there is no API or way right now, perhaps the way to customize this with GCS IAM [1] is that to use service account file. 

https://cloud.google.com/iam/docs/creating-managing-service-account-keys
```
gcloud iam service-accounts keys create \
    ~/key.json \
    --iam-account my-sa-123@my-project-123.iam.gserviceaccount.com
```

[1] - https://github.com/GoogleCloudPlatform/google-cloud-go#authorization